zetaclient/metrics/telemetry.go

```diff
@@ -67,6 +78,45 @@ func (t *TelemetryServer) GetIPAddress() string {
 	return t.ipAddress
 }
 
+func (t *TelemetryServer) GetLastStartTimestamp() time.Time {
+	t.mu.Lock()
+	defer t.mu.Unlock()
+	return t.lastStartTimestamp
+}
+
+func (t *TelemetryServer) SetLastScannedBlockNumber(chainName chains.ChainName, blockNumber uint64) {
+	t.mu.Lock()
+	t.lastScannedBlockNumber[chainName.String()] = blockNumber
+	LastScannedBlockNumber.WithLabelValues(chainName.String()).Set(float64(blockNumber))
+	t.mu.Unlock()
+}
+
+func (t *TelemetryServer) GetLastScannedBlockNumber(chainName chains.ChainName) uint64 {
+	t.mu.Lock()
+	defer t.mu.Unlock()
+	return t.lastScannedBlockNumber[chainName.String()]
+}
+
+func (t *TelemetryServer) SetCoreBlockNumber(blockNumber int64) {
+	t.mu.Lock()
+	t.lastCoreBlockNumber = blockNumber
+	LastCoreBlockNumber.Set(float64(blockNumber))
+	t.mu.Unlock()
+}
+
+func (t *TelemetryServer) GetCoreBlockNumber() int64 {
+	t.mu.Lock()
+	defer t.mu.Unlock()
+	return t.lastCoreBlockNumber
+}
+
+func (t *TelemetryServer) SetNumberOfUTXOs(numberOfUTXOs int) {
```

Comment:
NIT : maybe we can add comments ,for the functions. 
The functions itself are pretty straightforward but it wont hurt to add a comment for each ?